Source/WebCore/svg/SVGElement.cpp

```diff
@@ -284,15 +285,19 @@ Vector<WeakPtr<SVGResourceElementClient>> SVGElement::referencingCSSClients() co
     return copyToVector(m_svgRareData->referencingCSSClients());
 }
 
-void SVGElement::addReferencingCSSClient(SVGResourceElementClient& client)
+void SVGElement::addReferencingCSSClient(SVGResourceElementClient& client, const RenderElement& renderer)
```

Comment:
Why do we need to pass in "const RenderElement& renderer" here? where does it deviate from client->renderer()?

Reply:
There is no such method in SVGResourceElementClient. Would you prefer a virtual method for that in SVGResourceElementClient?

Reply:
No, I was wondering why you need to pass in the renderer at all, if it is client.renderer() that you pass around?

Reply:
This should be addressed by the masker PR, indeed it seems cleaner to just ask the client.

Reply:
Maybe you were looking at an older change?